kymatio/scattering3d/tests/test_torch_scattering3d.py

```diff
@@ -237,7 +237,7 @@ def test_solid_harmonic_scattering(device, backend):
     weights = np.ones((1, 1))
     sigma_gaussian = 3.
     sigma_0_wavelet = 3.
-    M, N, O, J, L = 128, 128, 128, 1, 3
+    M, N, O, J, L = 64, 64, 64, 1, 3
     grid = np.mgrid[-M // 2:-M // 2 + M, -N // 2:-N // 2 + N, -O // 2:-O // 2 + O]
```

Comment:
ok

Reply:
Note that the tolerance had to be lowered for this to still work, but I think it's fine.

Reply:
yes, this is more than fine